Tests/SwiftBuildTests/ConsoleCommands/ServiceConsoleTests.swift

```diff
@@ -144,7 +144,7 @@ extension Processes {
         #else
         Task<Void, Never>.detached {
             while true {
-                // We use getpgid() here to detect when the process has exited (it is not a child). Surprisingly, getpgid() is substantially faster than using kill(pid, 0) here because kill still returns success for zombies, and the service has been reparented to launchd.
+                // We use getpgid() here to detect when the process has exited (it is not a child). Surprisingly, getpgid() is substantially faster than using other methods which return success for zombies, and the service has been reparented to launchd.
```

Comment:
I'd keep this one since it's talking about the behavior of that specific API; just saying "other methods" loses information.